docs/support/faq/python-package-trouble.md

```diff
@@ -1,6 +1,40 @@
 # How to troubleshoot Python installation problems
 
-## Use `pip list -v` to find out where your packages are coming from
+## Run `which python3` to see if you are really in the environment you expect to be
+
+Some examples:
+
+```console
+$ module purge
+$ which python3
+/usr/bin/python3   # ← system Python
```

Comment:
A minor detail, but I'd suggest using the Bash built-in command `command` instead of the external `which`. When used with the `-V` option, a more verbose output is printed (as below). The `-v` option would print what `which` prints in this case.

```suggestion
$ command -V python3
python3 is /usr/bin/python3   # ← system Python
```